Softeq.XToolkit.Common/Helpers/Hashing/IHashCalculator.cs

```diff
@@ -0,0 +1,25 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+namespace Softeq.XToolkit.Common.Helpers.Hashing
+{
+    /// <summary>
+    /// Common interface for Hash function implementations
+    /// </summary>
+    public interface IHashCalculator
```

Comment:
`IHashCalculator` can be renamed to `IHashCodeCalculator`, to make an accent on the responsibility of abstraction

Reply:
Agreed